Dockerfile.py37

```diff
@@ -0,0 +1,12 @@
+FROM python:3.7-slim-buster
+
+MAINTAINER Flyte Team <users@flyte.org>
+LABEL org.opencontainers.image.source https://github.com/flyteorg/flytekit
+
+ARG $version
+
+RUN pip install awscli
+RUN pip install gsutil
+RUN pip install -U flytekit==$(echo version | sed 's#.*/v##')
+
```

Comment:
can we add an environment variable? `ENV FLYTE_INTERNAL_IMAGE=ghcr.io/flyteorg/flytekit:v<whatever the version is>`

Reply:
Or inject it in like so:
https://github.com/flyteorg/flytetools/blob/master/flytetester/Dockerfile#L55

And then set it while building:
https://github.com/flyteorg/flytetools/blob/master/flytetester/scripts/docker_build.sh#L39